src/paper/models.py

```diff
@@ -480,6 +480,16 @@ def votes_indexing(self):
             return [self.get_vote_for_index(vote) for vote in all_votes]
         return {}
 
+    def true_author_count(self):
+        registered_author_count = self.authors.count()
+        raw_author_count = len(self.raw_authors)
+
+        for author in self.raw_authors:
+            if self.authors.filter(first_name=author.get('first_name'), last_name=author.get('last_name')).exists():
```

Comment:
When I was working on elastic and had to run extract authors from this field I realized that it is not guaranteed that each author will be a `dict`. I ended up writing a function `parse_author_name` for this purpose. 

For your purpose though, you may want to do a check and make sure that the author is a `dict` similarly to what I do in `raw_authors_indexing` because sometimes it can be a string.

Reply:
@le-sun I want to get this cleaned before it's released then. 

Our raw authors should be in the same format regardless -- it should be either Null, an empty array, or an array of dicts